commandServer/routes.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+
+class BaseModel(object):
+
+    def get(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+    def post(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+    def put(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+    def delete(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
+
+
+ROUTES = {r'/': BaseModel,
```

Comment:
the function like a helper would take a URL path and point clients to needed views